<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QDialog" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audience Client Control Panel</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>581</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Refbox Connection</string>
   </property>
   <widget class="QLineEdit" name="txt_port">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_fps">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>34</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p FPS</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>581</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Reason for game stop</string>
   </property>
   <widget class="QPushButton" name="pushButton_pushing">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pushing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_repair">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Repair</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ballholding">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ball Holding</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_longdribble">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Long Dribble</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2robots">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2 Robots</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_illegaldefense">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Illegal Defense</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_illegalattack">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Illegal Attack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_popup">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Popup</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Show Message To Audience</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_msg">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>21</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>pushButton_pushing</zorder>
   <zorder>pushButton_repair</zorder>
   <zorder>pushButton_ballholding</zorder>
   <zorder>pushButton_longdribble</zorder>
   <zorder>pushButton_2robots</zorder>
   <zorder>pushButton_illegaldefense</zorder>
   <zorder>pushButton_illegalattack</zorder>
   <zorder>line</zorder>
   <zorder>btn_popup</zorder>
   <zorder>label_6</zorder>
   <zorder>txt_msg</zorder>
   <zorder>line_6</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Visualization Options</string>
   </property>
   <widget class="QPushButton" name="btn_flip">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip Sides</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fullscreen">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fullscreen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_windowed">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Windowed</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="select_screen">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="prefix">
     <string>Screen </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Show on</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>581</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Display Mode</string>
   </property>
   <widget class="QPushButton" name="pushButton_mode_black">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:checked{
	font: bold;
}</string>
    </property>
    <property name="text">
     <string>Black Screen</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mode_3d">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3D Field</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mode_image">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Image...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
